live-examples/js-examples/number/number-min-value.js

```diff
@@ -1,12 +1,12 @@
-function multiply(x, y) {
-  if (x * y < Number.MIN_VALUE) {
-    return 'Process as 0';
+function divide(x, y) {
+  if (x / y < Number.MIN_VALUE) {
+    return `Process as 0`;
```

Comment:
Don't use backticks here, but single quotes.

Reply:
I was testing out another approach, Sry I left it that way.
I'll make the changes and ping you.

Reply:
DONE!